src/test/test-sizeof.c

```diff
@@ -49,6 +49,12 @@ int main(void) {
         info(double);
         info(long double);
 
+        info(_Float128);
+        info(_Float64);
+        info(_Float64x);
+        info(_Float32);
+        info(_Float32x);
```

Comment:
will this break compilation on old gccs/glibcs?

Reply:
It will ;) There isn't an obvious define to check. ISO/IEC TS 18661-3 says that `__STDC_IEC_60559_TYPES__` should be defined by conforming implementations, but glibc doesn't have it. I added a check for `__HAVE_DISTINCT_FLOAT128X` which isn't very beautiful, but it should work. I assume that anything that doesn't have the type defined, will not have the define defined.